npm/janus.d.ts

```diff
@@ -130,33 +141,65 @@ declare namespace JanusJS {
 			[otherProps: string]: any;
 		};
 		jsep?: JSEP;
-		success?: Function;
-		error?: (error: any) => void;
+		success?: (data?: any) => void;
+		error?: (error: string) => void;
+	}
+	interface Publisher {
+		id: number; // <unique ID of active publisher #1>,
+		display: string; // "<display name of active publisher #1, if any>",
+		streams: MediaStream[];
 	}
 
+	interface WebRTCInfo {
+		bitrate: {
+			bsbefore: string | null;
+			bsnow: string | null;
+			timer: string | null;
+			tsbefore: string | null;
+			tsnow: string | null;
+			value: string | null;
+		};
+		dataChannel: Array<RTCDataChannel>;
+		dataChannelOptions: RTCDataChannelInit;
+
+		dtmfSender: string | null;
+		iceDone: boolean;
+		mediaConstraints: any;
+		mySdp: {
+			sdp: string;
+			type: string;
+		};
+		myStream: MediaStream;
+		pc: RTCPeerConnection;
+		receiverTransforms: {
+			audio: TransformStream;
+			video: TransformStream;
+		};
+		remoteSdp: string;
+		remoteStream: MediaStream;
+		senderTransforms: {
+			audio: TransformStream;
+			video: TransformStream;
+		};
+		started: boolean;
+		streamExternal: boolean;
+		trickle: boolean;
+		volume: {
+			value: number;
+			timer: number;
+		};
+	}
+	interface DetachOptions {
+		success?: () => void;
+		error?: (error: string) => void;
+		noRequest?: boolean;
+	}
 	interface PluginHandle {
 		plugin: string;
-		id: string;
+		opaqueId?: string;
```

Comment:
I can't find this parameter in the code

Reply:
It isn't, in fact: it can be passed to `createHandle`, but isn't stored anywhere, since it's passed to Janus right away when doing the `attach`.

Reply:
Ah, so this was mixed up with `id`. :+1: